UnityProject/Assets/Scripts/Items/Implants/Organs/BrainLaws.cs

```diff
@@ -0,0 +1,465 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Mirror;
+using Newtonsoft.Json;
+using Objects;
+using Objects.Research;
+using Systems.Ai;
+using UI.Core;
+using UI.Core.Action;
+using UI.Systems.MainHUD.UI_Bottom;
+using UnityEngine;
+
+public class BrainLaws : NetworkBehaviour, IActionGUI, ICheckedInteractable<HandActivate>
+{
+
+
+	public AiPlayer AiPlayer; //TODO refactor at some point
+
+	public ActionData ActionDataLawDisplay;
+
+	public ActionData ActionData => ActionDataLawDisplay;
+
+	// 	Law priority order is this:
+	//	0: Traitor/Malf/Onehuman-board Law
+	//  ##?$-##: HACKED LAW ##!£//#
+	//  ##!£//#: Ion Storm Law ##?$-##
+	//	Law 1: First Law
+	//	Law 2: Second Law
+	//	Law 3: Third Law
+	//	Law 4: Freeform
+	//	Higher laws (the law above each one) override all lower ones. Whether numbered or not, how they appear (in order) is the order of priority.
+
+	//Is sync'd manually to owner client so is accurate on owner client
+	//Tried to use sync dictionary from mirror but didnt work correctly, wouldnt sync the values correctly
+	private Dictionary<AiPlayer.LawOrder, List<string>> aiLaws = new Dictionary<AiPlayer.LawOrder, List<string>>();
+	public Dictionary<AiPlayer.LawOrder, List<string>> AiLaws => aiLaws;
+
+	private LightingSystem lightingSystem;
+
+	[SyncVar(hook = nameof(SynchronisedUpdate))]
+	private string SynchronisedLaws;
+
+	//Clientside only
+	private UI_Ai aiUi;
+
+	[SerializeField] private List<AiLawSet> defaultLawSets = new List<AiLawSet>();
+
+	private RegisterTile RegisterTile;
+
+	private void Awake()
+	{
+		RegisterTile = this.GetComponent<RegisterTile>();
+		if (aiUi == null)
+		{
+			aiUi = UIManager.Instance.displayControl.hudBottomAi.GetComponent<UI_Ai>();
+		}
+
+		if (lightingSystem == null)
+		{
+			lightingSystem = Camera.main.GetComponent<LightingSystem>();
+		}
+	}
+
+	#region AILinking
+
+	public bool WillInteract(HandActivate interaction, NetworkSide side)
+	{
+		if (!DefaultWillInteract.Default(interaction, side)) return false;
+
+		if (interaction.IsAltClick == false) return false;
```

Comment:
```cs
return interaction.IsAltClick;
```

Reply:
is Messy I don't like the return with statement, 
It's clear just to do the return true